torch/sparse/__init__.py

```diff
@@ -357,3 +358,102 @@ def sum(input: Tensor, dim: DimOrDims = None,
             [0, 0, 0, 0, 0],
             [0, 0, 0, 0, 0]])
 """)
+
+
+class check_sparse_tensor_invariants(object):
+    """A tool to control checking sparse tensor invariants.
+
+The following options exists to manage sparsr tensor invariants
+checking in sparse tensor construction:
+
+1. Using a context manager::
+
+     with torch.sparse.check_sparse_tensor_invariants(enable=True):
+         run_my_model()
+
+2. Using a procedural approach::
+
+    prev_checks_enabled = torch.sparse.check_sparse_tensor_invariants.is_enabled()
+    torch.sparse.check_sparse_tensor_invariants.enable()
+
+    run_my_model()
+
+    if not prev_checks_enabled:
+        torch.sparse.check_sparse_tensor_invariants.disable()
+
+3. Using function decoration::
+
+    @torch.sparse.check_sparse_tensor_invariants(enable=True)
```

Comment:
Same here. I think it makes for a better example if we remove the explicit `enable=True`.

Reply:
Fixed.